cloudevents/tests/test_http_events.py

```diff
@@ -0,0 +1,63 @@
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from sanic import Sanic
+from sanic import response
+
+from cloudevents.sdk.http_events import Event
+
+from cloudevents.tests.data import http_event_test_headers as test_headers
+from cloudevents.tests.data import http_event_ce_json as test_data
+
+import requests
+import json
+
+app = Sanic(__name__)
+
+
+def post(url, headers, json):
+    return app.test_client.post(url, headers=headers, data=json)
+
+
+@app.route("/event", ["POST"])
+async def echo(request):
+    event = Event(dict(request.headers), request.body)
+    return response.text(event.data, headers=event.headers)
+
+
+def test_invalid_binary_headers():
```

Comment:
Add a comment as to what this actually tests. I don't know exactly where we check for invalid headers. Does the `Event` constructor throw a TypeError if there is an invalid event?

Reply:
Yeah. I added an explanatory comment just now